tests/vxlan/bfd_sniffer.py

```diff
@@ -0,0 +1,46 @@
+from scapy.all import *
```

Comment:
Please reformat this file according to PEP8.

Reply:
@ihorchekh , we are planning to remove this file, as it is not needed for the tests.